From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Mon, 19 Aug 2024 20:19:09 -0500
Subject: [PATCH] Work if invoked as libtoolize

On Serpent we install a libtoolize symlink, so add detection to properly handle if slibtool is invoked like that.

---
 src/driver/slbt_amain.c | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/src/driver/slbt_amain.c b/src/driver/slbt_amain.c
index e0f4241..f9e53ea 100644
--- a/src/driver/slbt_amain.c
+++ b/src/driver/slbt_amain.c
@@ -152,6 +152,9 @@ int slbt_main(char ** argv, char ** envp, const struct slbt_fd_ctx * fdctx)
 	else if (!(strcmp(program,"slibtoolize")))
 		flags |= SLBT_DRIVER_MODE_STOOLIE;
 
+	else if (!(strcmp(program,"libtoolize")))
+		flags |= SLBT_DRIVER_MODE_STOOLIE;
+
 	/* debug */
 	if (!(strcmp(program,"dlibtool")))
 		flags |= SLBT_DRIVER_DEBUG;
